plenum/test/monitoring/test_invalid_reqs_in_monitor.py

```diff
@@ -0,0 +1,38 @@
+import functools
+
+from plenum.common.exceptions import InvalidClientMessageException, RequestRejectedException
+from plenum.test.helper import sdk_send_random_and_check
+
+
+REQ_COUNT = 10
+
+
+def randomDynamicValidation(self, req):
+    if list(self.requests.keys()).index(req.key) % 2:
+        raise InvalidClientMessageException('aaaaaaaa',
+                                             req.reqId,
+                                             "not valid req")
+
+def check_count_reqs(nodes):
+    for node in nodes:
+        reqs_count = set([c for c, _ in node.monitor.numOrderedRequests])
+        assert len(reqs_count) == 1
+
+
+def test_invalid_reqs(looper,
```

Comment:
Let's make the test more predictable:
send 1 valid and 2 invalid reqs, and check that `numOrderedRequests` in monitor is equal to 3.
```
sdk_send_random_and_check(...., 1)
<patch dynamic validation ot always return false>
with pytest,raises(RequestRejectedException)
    sdk_send_random_and_check(...., 1)
with pytest,raises(RequestRejectedException)
    sdk_send_random_and_check(...., 1)
check_count_reqs(nodes, 3)
```